src/components/News/NewsFilter/NewsFilter.tsx

```diff
@@ -0,0 +1,138 @@
+            <circle cx="4" cy="7" r="1" fill="white" stroke="#454B50" strokeWidth="2" />
+            <circle cx="8" cy="2" r="1" fill="white" stroke="#454B50" strokeWidth="2" />
+          </svg>
+          <Dropdown
+            data-testid="dropdown-button-filter"
+            dropdownButtonText={dropdownFilter ?? "Filter"}
+            className="rounded-md py-1 pl-8 pr-2 border border-gray-300 bg-white"
+            classNameShared="w-full max-w-xs"
+          >
+            <DropdownItem
+              data-testid="show-all"
+              onClick={() => {
+                setDropdownFilter(undefined);
+              }}
+            >
+              Show all
+            </DropdownItem>
+            {Object.entries(NewsTag).map((entry, index) => {
+              const key = entry[0];
+              const value = entry[1];
+
+              return (
+                <DropdownItem
+                  key={index}
+                  data-testid={key}
+                  onClick={() => {
+                    setDropdownFilter(NewsTag[key as keyof typeof NewsTag]);
+                  }}
+                >
+                  {value}
+                </DropdownItem>
+              );
+            })}
+          </Dropdown>
+        </div>
+      </div>
+      <div className="w-full lg:w-60">
+        <div className="relative">
+          <svg
+            className="absolute top-1/2 -mt-1 left-3 z-20"
+            width="11"
+            height="10"
+            viewBox="0 0 11 10"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path d="M1 5L7 5" stroke="#454B50" strokeWidth="2" strokeLinecap="round" />
+            <path d="M1 1H10" stroke="#454B50" strokeWidth="2" strokeLinecap="round" />
+            <path d="M1 9H4" stroke="#454B50" strokeWidth="2" strokeLinecap="round" />
+          </svg>
+          <Dropdown
+            data-testid="dropdown-button-sort"
+            dropdownButtonText={dropdownSort ?? "Sort By"}
+            className="rounded-md py-1 pl-8 pr-2 border border-gray-300 bg-white"
+            classNameShared="w-full max-w-xs"
+          >
+            <DropdownItem
+              onClick={() => {
+                setDropdownSort(undefined);
+              }}
+            >
+              Clear
+            </DropdownItem>
+            {Object.entries(NewsSort).map((entry, index) => {
+              const key = entry[0];
+              const value = entry[1];
+
+              return (
+                <DropdownItem
+                  data-testid={value}
+                  key={index}
+                  onClick={() => {
+                    setDropdownSort(NewsSort[key as keyof typeof NewsSort]);
+                  }}
+                >
+                  {value}
+                </DropdownItem>
+              );
+            })}
+          </Dropdown>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
both `filter` and `sort by` seem to share alot in common.

i would approach it by having another component that expects
1) icon
2) 1 call to action function

so that when i read this component, i immediately know which one sorts and which one filters.

but since we are not going to reuse this, i'm fine